skeleton/common/tsconfig.json__if_typescript

```diff
@@ -32,10 +32,8 @@
     "types": ["node", "jest"],
     // @endif
 
-    // @if feat.webpack && feat['postcss-typical'] && feat.protractor
```

Comment:
I think you accidentally removed some lines?

Reply:
Ok, I saw your commit message. But we did this conditional typeRoots to do minimum impact for users of lerna (where `"./node_modules/@types"` does not work).